docker/Dockerfile.kubectl

```diff
@@ -1,7 +1,11 @@
-# hadolint ignore=DL3006
-FROM istionightly/base_debug
 # Image for post install jobs
 
+# Version is the base image version from the TLD Makefile
+ARG ISTIO_VERSION=latest
+
+# The following section is used as base image if BASE_DISTRIBUTION=default
+FROM docker.io/sdake/base_debug:${ISTIO_VERSION} as default
+
```

Comment:
I am assuming that we will be using a different repo for the base_debug image instead of sdake?

Reply:
ya, just for testing, hence the WIP marking.

This would be `FROM docker.io/istio/base_debug:${ISTIO_VERSION}` or whatever source people want to put the base_debug iamge.

The "cronjob" is not actually cronjobbing the generation of the base_debug image.  See docker.mk for more details.